README.md

```diff
@@ -0,0 +1,24 @@
+# Data Plane Gateway
+
+[Gazette](https://github.com/gazette/core) exposes several GRPC services we'd like to provide broader access to.
+1. We'd like to expose REST versions of these services.
+2. We'd like to enforce authentication and authorization. Coming soon:tm:.
+
+![Data Plane Gateway](docs/data-plane-gateway-v1.png)
+
+## Rest Gateway
+
+We use the [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway/) project to generate rest handlers from the protobuf definitions found in Gazette. We also generate swagger documentation which we then use to generate a typescript client.
+
+#### Components
+
+* `{broker,consumer}_service.yaml` define which RPCs are exposed as rest handlers.
```

Comment:
> exposed as rest handlers.

The acronym _rest_ should be capitalized. This comment applies to line 17 and 18 as well. Calling this out for general awareness and record.